src/index.ts

```diff
@@ -32,6 +32,15 @@ void (async () => {
 
 	const config = await startup();
 
+	//if addrepo is passed then deploy from repository url
+	if (args['addrepo']) {
+		try {
+			await deployFromRepository(config, plan);
+		} catch (e) {
+			console.error(e);
+		}
+	}
+
 	//if workdir is passed call than deploy using package
 	if (args['workdir']) {
```

Comment:
This will always result to true.

Reply:
no it is older commit see latest one

Reply:
```suggestion
	if (args['workdir']) {
```


Isn't this in your latest commit? workdir have a default value, will be true always

Reply:
see this
https://github.com/itsritiksingh/deploy/blob/40c32bbb87ed6828401a8b69e102720aabd3021f/src/index.ts#L45

it is check at the end